app/services/SortStatementsService.scala

```diff
@@ -64,8 +64,9 @@ class SortStatementsService @Inject()() {
     VatCertificatesForEori(historicEori, current, requested)
   }
 
-  def sortPostponedVatStatementsForEori(historicEori: EoriHistory, postponedVatStatementsFile: Seq[PostponedVatStatementFile])
-                                (implicit messages: Messages): PostponedVatStatementsForEori = {
+  def sortPostponedVatStatementsForEori(historicEori: EoriHistory,
```

Comment:
Could you please format like below to align the implicit parameters properly?

  def sortPostponedVatStatementsForEori(historicEori: EoriHistory,
                                        postponedVatStatementsFile: Seq[PostponedVatStatementFile])
                                       (implicit messages: Messages): PostponedVatStatementsForEori = {